app/assets/templates/static_pages/index.html.erb

```diff
@@ -0,0 +1,125 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <!-- Brand and toggle get grouped for better mobile display -->
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
+        <span class="sr-only">
+          Toggle navigation
+        </span>
+        <span class="icon-bar">
+        </span>
+        <span class="icon-bar">
+        </span>
+        <span class="icon-bar">
+        </span>
+      </button>
+      <a class="navbar-brand myLogo" href="/#/">
+        <img src="<%= asset_path('logo.png') %>" width="100" height="55" >
+      </a>
+
+      <!-- Collect the nav links, forms, and other content for toggling -->
+      <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+        <ul class="nav navbar-nav navbar-left">
+        </ul>
+        <ul class="nav navbar-nav navbar-right">
+          <li>
+          </li>
+        </ul>
+      </div>
+      <!-- /.navbar-collapse -->
+    </div>
+   <!-- /.container-fluid -->
+  </div>
+ </nav>
+
+
+<div class="container-fluid" id="landing">
+  <div class="col-xs-5 col-md-2"></div>
+
+  <div class="col-xs-8 col-md-8">
+    <font color="black">
+      <h3>
+        <strong>
+          Welcome to Mech-Assist:
+          <br>
+          Improve your Chinese Literacy with
+          <i>
+            <ins>
+              Mechanical Assistance
+            </ins>
+          </i>
+          !
+        </br>
+      </strong>
+      </h3>
+
+
+          <h4>
+            Introduction
+          </h4>
+
+          <p>To learn how to read and write chinese characters, the most important thing is to understand each character's context, its meaning, and how it is associated with other characters.
+          </p>
+
+          <p>There are many words that are hard to memorize and require you to check to dictionary for pronounciation, pinyin, and english translation.
+          </p>
+
+          <p>Since China has adopted simplified characters for all formal writing, I recommend learning both traditional and simplified characters.
+          </p>
+
+          <p>Mecha-Assist's website tries to provide features to help facilitate your learning. Memorization happens not by brute force but by reasoning and practice.
+          </p>
+
+        <h4>
+          Vocabulary
+        </h4>
+
+          <ol>
+            <li>Insert chinese article of your choice.</li>
+
+            <img src="http://www.chinese-community-center.org/images%5Ccarticle1.jpg" height="25%" width="25%" />
```

Comment:
you should add all external images to your `/assets/images` folder and then include them with `image_tag`. That way, if the image is not available externally anymore, your app won't break